resources/js/Pages/SurveyResults/Container.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <app-layout>
+
+        <template #header>
+            <h2 class="font-semibold text-xl text-gray-800 leading-tight">
+                Umfrage Resultate
+            </h2>
+        </template>
+
+            <div class="container mx-auto pt-5">
+                <div class="w-full bg-white rounded shadow-lg p-8 m-4">
+                    <div class="flex space-x-4 flex-col">
+                        <SurveyDisplayer :name="survey['survey_name']" :creator="survey.user?.name">
```

Comment:
try to Use the Object Syntax. I Dont know if there are any Tecnical Differneces but it is Cleaner if its the Same Everywhere.
survey['survey_name']  -> survey.survey_name